src/scripts/main.js

```diff
@@ -1,3 +1,48 @@
 'use strict';
 
-// write your code here
+let bodyDoc;
+let mateLogo;
+
+if (document.querySelector('body') !== null) {
+  bodyDoc = document.querySelector('body');
+}
+
+if (document.querySelector('.logo') !== null) {
+  mateLogo = document.querySelector('.logo');
```

Comment:
Similarly to the 'body' element, you are querying the DOM for the '.logo' element twice. Store the result in a variable and check its existence to improve performance.